data_structures/queue/priority_queue.py

```diff
@@ -0,0 +1,248 @@
+"""
+Pure Python implementation of Priority Queue using lists
+"""
+
+
+class FixedPriorityQueue:
+    """
+    In a Priority Queue the elements are entred as an when the come
+    But while removing or deleting an element the highest priority element is deleted in FIFO fashion
+    Here the lowest integer has the highest priority.
+    Example:
+    priority(0) > priority(5)
+    priority(16) > priority(32)
+    Here as an example I have taken only 3 priorities viz. 0, 1, 2
+    0 Being the highest priority and 2 being the lowest
+    You can change the priorities as per your need
+
+    Examples
+    >>>FPQ = FixedPriorityQueue()
+    >>>FPQ.enqueue(0, 10)
+    >>>FPQ.enqueue(1, 70)
+    >>>FPQ.enqueue(0, 100)
+    >>>FPQ.enqueue(2, 1)
+    >>>FPQ.enqueue(2, 5)
+    >>>FPQ.enqueue(1, 7)
+    >>>FPQ.enqueue(2, 4)
+    >>>FPQ.enqueue(1, 64)
+    >>>FPQ.enqueue(0, 128)
+    >>>FPQ.print_queue()
+    Priority 0: [10, 100, 128]
+            Priority 1: [70, 7, 64]
+            Priority 2: [1, 5, 4]
+    >>>print(FPQ.dequeue())
+    10
+    >>>print(FPQ.dequeue())
+            100
+    >>>print(FPQ.dequeue())
+    128
+    >>>print(FPQ.dequeue())
+    70
+    >>>print(FPQ.dequeue())
+    7
+    >>>FPQ.print_queue()
+    Priority 0: []
+    Priority 1: [64]
+    Priority 2: [1, 5, 4]
+    >>>print(FPQ.dequeue())
+    64
+    >>>print(FPQ.dequeue())
+    1
+    >>>print(FPQ.dequeue())
+    5
+    >>>print(FPQ.dequeue())
+    4
+    >>>print(FPQ.dequeue())
+    Traceback (most recent call last):
+      File "Sample.py", line 117, in <module>
+            print(FPQ.dequeue())
+      File "Sample.py", line 58, in dequeue
+            raise Exception("Under Flow!")
```

Comment:
```suggestion
      ...
```
Read the doctest section in the Python docs to see why we use the three dots. (ellipsis)